libraries/AP_AHRS/AP_AHRS_DCM.h

```diff
@@ -48,14 +48,21 @@ class AP_AHRS_DCM : public AP_AHRS
         _last_consistent_heading(0),
         _imu1_weight(0.5f),
         _last_failure_ms(0),
-        _last_startup_ms(0)
+        _last_startup_ms(0),
+        _EKF_home_present(false)
     {
         _dcm_matrix.identity();
 
         // these are experimentally derived from the simulator
         // with large drift levels
         _ki = 0.0087f;
         _ki_yaw = 0.01f;
+
+        // Initialize _EKF_home
+        _EKF_home.options    = 0;
```

Comment:
This should be called EKF origin, rather then EKF home. There is a conceptual difference between home and origin within the EKF that should be preserved.